refarch-integrations/refarch-dms-integration/refarch-dms-integration-fabasoft/refarch-dms-integration-fabasoft-mock/src/main/java/de/muenchen/refarch/integration/dms/fabasoft/mock/FabasoftClienFactory.java

```diff
@@ -8,9 +8,11 @@
 /**
  * Configuration of the Webservice Client.
  */
-public class FabasoftClienFactory {
+public final class FabasoftClienFactory {
+    private FabasoftClienFactory() {
+    }
```

Comment:
Should we add throwing a Exception here as instantiating (even inside this class) is not allowed?

Reply:
Think that just makes the code more unreadable and pmd should raise some warning if used.